scripts/github_runner/setup_cri_test_env.sh

```diff
@@ -22,31 +22,28 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-PWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-$PWD/setup_system.sh
-$PWD/create_devmapper.sh
-
-# install Protocol Buffer Compiler
-PROTO_VERSION=3.11.4
-if [ ! -f "protoc-$PROTO_VERSION-linux-x86_64.zip" ]; then
-    wget -c "https://github.com/google/protobuf/releases/download/v$PROTO_VERSION/protoc-$PROTO_VERSION-linux-x86_64.zip"
-    sudo unzip -u "protoc-$PROTO_VERSION-linux-x86_64.zip" -d /usr/local
-    rm "protoc-$PROTO_VERSION-linux-x86_64.zip"
+set -Eeuo pipefail
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+if (( $# != 1)); then
+    echo "Invalid number of parameters"
+    echo "USAGE: setup_cri_test_env.sh <sandbox>"
+    exit 1
 fi
 
-# Necessary for containerd as container runtime but not docker
-sudo modprobe overlay
-sudo modprobe br_netfilter
+SANDBOX=$1
+ROOT="./../.."
+
+$ROOT/scripts/cloudlab/start_onenode_vhive_cluster.sh "$SANDBOX"
+sleep 30s
 
-# Set up required sysctl params, these persist across reboots.
-sudo tee /etc/sysctl.d/99-kubernetes-cri.conf <<EOF
-net.bridge.bridge-nf-call-iptables  = 1
-net.ipv4.ip_forward                 = 1
-net.bridge.bridge-nf-call-ip6tables = 1
-EOF
+KUBECONFIG=/etc/kubernetes/admin.conf sudo $ROOT/scripts/setup_zipkin.sh
 
-sudo sysctl --system
-sudo swapoff -a
+source etc/profile && go run $ROOT/examples/registry/populate_registry.go -imageFile $ROOT/examples/registry/images.txt
```

Comment:
we should not have these untill we update the ubuntu version in vhive profile of cloudlab, local-registry tests can be skipped for now. I've create a separate issue for that you can comment this line and reference it here

Reply:
@anshalshukla let's merge the PR merging the local-registry removal before this PR and rebase.

Reply:
Opened #710 to close #703.